### DISCUSSION TITLE
Does SendPortal make attempts to distinguish between bots/link scanners and Real Clicks/Opens

### DISCUSSION THREAD

**Discussion by paulhouselander** [mettle/sendportal#296] [General] [answered]
I was looking for some email marketing software that helped track opens/clicks.

Does SendPortal make any attempts to filter out non-human activity's such as automated link scanners etc...

Thanks

Paul

**Comment by JonoB** [comment, marked as answer]
We rely completely on the email service provider (Mailgun, Sendgrid, Postmark, etc)
